tests/zfs-tests/tests/functional/cli_root/zdb/zdb_002_pos.ksh

```diff
@@ -0,0 +1,51 @@
+#!/bin/ksh
```

Comment:
GitHub doesn't appear to show the permission bits for this new file, please just make sure it's executable like the other scripts or the automated testing will skip it.